src/test/java/com/artipie/rpm/meta/XmlPrimaryTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Tests for {@link XmlPrimary}.
+ *
+ * @since 1.0
+ */
+public final class XmlPrimaryTest {
+    // @todo #83:30min This test only creates a temporary file for XmlPrimary,
+    //  now some assertion should be added to verify that this class
+    //  can write `primary.xml` file correctly. The example of primary can
+    //  be found at test resources. If #86 is fixed then remove the DisabledOnOs
+    //  annotation.
+    @Test
+    @DisabledOnOs(OS.WINDOWS)
+    public void writesFile(@TempDir final Path temp) throws Exception {
```

Comment:
@victornoel I understand that this test is disabled because #86, but why don't make the test complete? Isn't that the objective of this PR?

Reply:
@victornoel it is the objective of the todo yes, but I couldn't do everything in the allotted time, in particular I have no idea how to verify the content of the file: I believe writing the particular assertion for this test is going to take the time of a new ticket, hence the todo above. If you have some idea I can add them to the todo actually.

Reply:
@paulodamaso it is the objective of the todo yes, but I couldn't do everything in the allotted time, in particular I have no idea how to verify the content of the file: I believe writing the particular assertion for this test is going to take the time of a new ticket, hence the todo above. If you have some idea I can add them to the todo actually.